config/xsl/loadreport/util/timer-section.xsl

```diff
@@ -7,6 +7,17 @@
         <xsl:param name="directory"/>
         <xsl:param name="runtimeIntervalsNode"/>
         <xsl:param name="type"/>
+        
+        <h3 class="no-print">Summary</h3>
```

Comment:
According to feedback, the Summary header is not really needed and should be removed.

Reply:
Please remove the heading altogether. No need to have it as comment in the template.